scout/server/blueprints/cases/templates/cases/chanjo2_form.html

```diff
@@ -2,6 +2,7 @@
 
   {% set form_name = "chanjo2_" ~ panel_name ~ "_" ~ report_type %}
   {% set form_action = config.CHANJO2_URL+"/"+report_type %}
+  {% set access_token = session.token_response.access_token if session.token_response else '' %}
```

Comment:
Access token is sent regardless of chanjo2 settings.

It is validated in chanjo2 only if chanjo2 has the right settings:

<img width="1083" alt="image" src="https://github.com/user-attachments/assets/ef98b040-89ea-4843-85bf-0146a0d4cf99" />